critiquebrainz/frontend/templates/place/entity.html

```diff
@@ -93,10 +94,8 @@ <h4 style="margin-bottom:0;">{{ _('Reviews') }}</h4>
   <div class="col-md-3">
     <h4>{{ _('Place information') }}</h4>
       {% if avg_rating %}
```

Comment:
This checking can also be moved inside the `show_avg_rating` macro. And simply pass `avg_rating` to it. Make usage as simple as possible.